assets/quick-buy.css

```diff
@@ -0,0 +1,188 @@
+.quick-buy-modal {
```

Comment:
I'll do a follow up PR to combine modal styles in a single component. 

Decided to skip this here to maintain the scope smaller.